scripts/send_erpc_entrants_email.py

```diff
@@ -0,0 +1,108 @@
+import sys
+import logging
+
+from website.app import setup_django
+setup_django()
+
+from website import settings
+from scripts import utils as script_utils
+from osf.models import MetaSchema, Contributor, Registration
+from framework.email.tasks import send_email
+
+logger = logging.getLogger(__name__)
+
+
+EMAIL_TEXT = """
+
+Dear ERPC participants,
+
+Thanks so much for registering a design for the Election Research Preacceptance Competition. We are writing to clarify the competition rules now that the ANES data have been publicly released.
+
+Even if you have not yet submitted an article based on your design, you can still publish your findings and win a cash prize as long as your paper includes an analysis of your preregistered design and clearly indicates any difference between the submitted design and any subsequent exploratory analyses. At this point in the competition, preacceptance is no longer a necessary condition for being published at a participating journal or winning a prize.
+
+We have worked with all of the participating editors to learn about how they wish to receive ERPC manuscripts going forward. Here are their preferences:
+
+-Political Behavior and Public Opinion Quarterly would like you to submit ERPC manuscripts *without* the data or analyses (in other words, they want to review the article under preacceptance conditions).
+
+-American Journal of Political Science, American Politics Research, Political Analysis, and Political Science Quarterly prefer that you submit ERPC manuscripts *with* the data or analyses (in other words, they want to review manuscripts than include your pre-registered designs, but not under conditions of preacceptance).
+
+-Finally, American Political Science Review and State Politics and Policy Quarterly will accept your ERPC manuscript for review with or without the data.
+
+(Political Science Research and Methods has not yet indicated its preferences; please contact the editor for clarification before submitting an article to them.)
```

Comment:
Also had to remove the parentheses here to prevent Gmail from truncating the message.